scalafmt-tests/src/test/resources/trailing-commas/trailingCommasMultiple.stat

```diff
@@ -147,3 +147,93 @@ object Foo {
       foo())
   }
 }
+<<< #2755 one, comma
+maxColumn = 80
+runner.parser = source
+===
+import foo.{
+  b,
+}
+class foo(
+  b: String,
+) {
+  def method(
+    b: String,
+  ) = ???
+  method(
+    b,
+  )
+}
+>>>
+import foo.{b}
+class foo(
+    b: String
+) {
+  def method(
+      b: String
+  ) = ???
+  method(
+    b
+  )
+}
+<<< #2755 two, comma
+maxColumn = 80
+runner.parser = source
+===
+import foo.{a,
+  b,
+}
+class foo(a: String,
+  b: String,
+) {
+  def method(a: String,
+    b: String,
+  ) = ???
+  method(a,
+    b,
+  )
+}
+>>>
+import foo.{a, b}
+class foo(a: String, b: String) {
```

Comment:
Shouldn't this actually have `,`  ? That seems to be happening with https://scalameta.org/scalafmt/docs/configuration.html#trailing-commas-multiple

Reply:
if you take a look at the second commit separately, you'll see the test that is fixed by this pr.